resources/install.rb

```diff
@@ -79,6 +79,10 @@
     make_cmd = "make TARGET=#{new_resource.source_target_os}"
     make_cmd << " CPU=#{new_resource.source_target_cpu}" unless new_resource.source_target_cpu.nil?
     make_cmd << " ARCH=#{new_resource.source_target_arch}" unless new_resource.source_target_arch.nil?
+    if node['platform_family'] == 'rhel' && node['platform_version'].to_i < 7
+      make_cmd << ' USE_RT=1'
+      make_cmd << ' USE_CRYPT_H=1'
+    end
```

Comment:
I know I am being pedantic, but could this be moved to the end, before extra_config?

Reply:
I just pushed that change up